test/test.py

```diff
@@ -800,6 +808,22 @@ def test_multiple_requests(self):
 		self.assertEqual(len(trace_ids), 1)
 		self.assertEqual(len(parent_ids), 3)
 
+	def test_multiple_requests_with_illegal_traceparent(self):
```

Comment:
I feel this is very specific to the test implementation, the fact that all 3 requests will have the same trace_id is not part of the specs in my opinion. We just need to check that they have a valid trace_id in my opinion, if an implementation of the test will create a new root span (new trace id) for every request I don't think that is necessary bad.

Reply:
@bogdandrutu you're right.
It seems that we've only mentioned "restart trace" here https://github.com/w3c/trace-context/blob/master/spec/20-http_header_format.md#versioning-of-traceparent.

@SergeyKanzhelev what's your opinion?

Reply:
Yes, thinking about it - @bogdandrutu you are correct. We never ask for both - incoming and outgoing requests to be recorded necessarily. So if you only monitor outgoing calls - this test will not hold. @reyang would you mind just checking for correctness?

Reply:
Test cases updated.